x-pack/plugins/task_manager/server/plugin.ts

```diff
@@ -156,6 +159,7 @@ export class TaskManagerPlugin
       fetch: (opts: SearchOpts): Promise<FetchResult> => taskStore.fetch(opts),
       get: (id: string) => taskStore.get(id),
       remove: (id: string) => taskStore.remove(id),
+      deleteTaskIfItExists: (id: string) => deleteTaskIfItExists(taskStore, id),
```

Comment:
can we rename `deleteTaskIfItExists` to `removeIfExists` ?
All these API calls are for tasks, so this should only include `Task` if the others do too.
Best just to omit in all of them I think :)